Tests/ClientTests/Coordinators/BrowserCoordinatorTests.swift

```diff
@@ -145,6 +145,42 @@ final class BrowserCoordinatorTests: XCTestCase {
         XCTAssertNotNil(screenshotService.screenshotableView)
     }
 
+    func testHandleSearchQuery_returnsTrue() {
+        let subject = createSubject()
+        let result = subject.handle(route: .searchQuery(query: "test query"))
+        XCTAssertTrue(result)
```

Comment:
Could we assert the BVC method is called? We might need to inject the BVC into the BrowserCoordinator (instead of creating it in the init) so we can have a mockBVC for our tests to assert that.

Reply:
Sure, mocking bvc will be an interesting thing to do.